TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -346,7 +346,11 @@ private void TryPopModal()
             if (blurEffect != null && UIView.ModalInputCount() == 0) {
                 blurEffect.eventPositionChanged -= OnBlurEffectPositionChange;
                 blurEffect.eventZOrderChanged -= OnBlurEffectZOrderChange;
-                ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(1f, 0f, 0.7f, EasingType.CubicEaseOut), delegate () { blurEffect.Hide(); });
+                ValueAnimator.Animate(
```

Comment:
Another delegate issue?

Reply:
Not an issue, delegates are gone and replaced with lambdas, this is intended and does not change the logic.